exchangis-datasource/exchangis-datasource-streamis/src/main/scala/com/webank/wedatasphere/exchangis/datasource/streamis/ExchangisStreamisRemoteClient.scala

```diff
@@ -0,0 +1,167 @@
+package com.webank.wedatasphere.exchangis.datasource.streamis
+
+import com.webank.wedatasphere.exchangis.datasource.core.ExchangisDataSourceConfiguration
+import com.webank.wedatasphere.linkis.datasource.client.impl.{LinkisDataSourceRemoteClient, LinkisMetaDataRemoteClient}
+import com.webank.wedatasphere.linkis.datasource.client.request._
+import com.webank.wedatasphere.linkis.datasource.client.response._
+import com.webank.wedatasphere.linkis.datasourcemanager.common.domain.{DataSource, DataSourceType}
+import com.webank.wedatasphere.linkis.httpclient.dws.authentication.StaticAuthenticationStrategy
+import com.webank.wedatasphere.linkis.httpclient.dws.config.{DWSClientConfig, DWSClientConfigBuilder}
+
+import java.lang
+import java.util.concurrent.TimeUnit
+
+object ExchangisStreamisRemoteClient {
+  //Linkis Datasource Client Config
+  val serverUrl: String = ExchangisDataSourceConfiguration.SERVER_URL.getValue
+  val connectionTimeout: lang.Long = ExchangisDataSourceConfiguration.CONNECTION_TIMEOUT.getValue
+  val discoveryEnabled: lang.Boolean = ExchangisDataSourceConfiguration.DISCOVERY_ENABLED.getValue
+  val discoveryFrequencyPeriod: lang.Long = ExchangisDataSourceConfiguration.DISCOVERY_FREQUENCY_PERIOD.getValue
+  val loadbalancerEnabled: lang.Boolean = ExchangisDataSourceConfiguration.LOAD_BALANCER_ENABLED.getValue
+  val maxConnectionSize: Integer = ExchangisDataSourceConfiguration.MAX_CONNECTION_SIZE.getValue
+  val retryEnabled: lang.Boolean = ExchangisDataSourceConfiguration.RETRY_ENABLED.getValue
+  val readTimeout: lang.Long = ExchangisDataSourceConfiguration.READ_TIMEOUT.getValue
+  val authTokenKey: String = ExchangisDataSourceConfiguration.AUTHTOKEN_KEY.getValue
+  val authTokenValue: String = ExchangisDataSourceConfiguration.AUTHTOKEN_VALUE.getValue
+  val dwsVersion: String = ExchangisDataSourceConfiguration.DWS_VERSION.getValue
+
+
+  //  val clientConfig = DWSClientConfigBuilder.newBuilder()
+  //    .addServerUrl(serverUrl)
+  //    .connectionTimeout(connectionTimeout)
+  //    .discoveryEnabled(discoveryEnabled)
+  //    .discoveryFrequency(1,TimeUnit.MINUTES)
+  //    .loadbalancerEnabled(loadbalancerEnabled)
+  //    .maxConnectionSize(maxConnectionSize)
+  //    .retryEnabled(retryEnabled)
+  //    .readTimeout(readTimeout)
+  //    .setAuthenticationStrategy(new StaticAuthenticationStrategy())
+  //    .setAuthTokenKey(authTokenKey)
+  //    .setAuthTokenValue(authTokenValue)
+  //    .setDWSVersion(dwsVersion)
+  //    .build()
+
+  val clientConfig: DWSClientConfig = DWSClientConfigBuilder.newBuilder()
```

Comment:
Should we provide the method to custom client configuration ? I think the parameters of configuration should not be fixed.